src/www/app/outline_server_repository/outline_server_repository.spec.ts

```diff
@@ -16,35 +16,38 @@ import {InMemoryStorage} from '../../../infrastructure/memory_storage';
 import {ServerIncompatible, ServerUrlInvalid, ShadowsocksUnsupportedCipher} from '../../model/errors';
 import {EventQueue, ServerAdded, ServerForgetUndone, ServerForgotten, ServerRenamed} from '../../model/events';
 
-import {ShadowsocksConfig} from '../config';
+import {ShadowsocksSessionConfig} from './shadowsocks_session_config';
 import {FakeNativeNetworking} from '../fake_net';
 import {FakeOutlineTunnel} from '../fake_tunnel';
 
 import {OutlineServerRepository, ServersStorageV0, ServersStorageV1} from '.';
 import {OutlineServer} from './server';
-import {shadowsocksConfigToAccessKey} from './access_key_serialization';
+import {shadowsocksSessionConfigToAccessKey} from './access_key_serialization';
 
 // TODO(alalama): unit tests for OutlineServer.
 
 describe('OutlineServerRepository', () => {
-  const CONFIG_0: ShadowsocksConfig = {
+  const CONFIG_0: ShadowsocksSessionConfig = {
     host: '127.0.0.1',
     port: 1080,
     password: 'test',
     method: 'chacha20-ietf-poly1305',
-    name: 'fake server 0',
+    tag: 'fake server 0',
   };
 
-  const CONFIG_1: ShadowsocksConfig = {
+  const CONFIG_1: ShadowsocksSessionConfig = {
     host: '10.0.0.1',
     port: 1089,
     password: 'test',
     method: 'chacha20-ietf-poly1305',
-    name: 'fake server 1',
+    tag: 'fake server 1',
   };
 
   it('loads V0 servers', () => {
-    const storageV0: ServersStorageV0 = {'server-0': CONFIG_0, 'server-1': CONFIG_1};
+    const storageV0: ServersStorageV0 = {
+      'server-0': {...CONFIG_0, name: CONFIG_0.tag},
```

Comment:
This should keep the v0 format for a valid test. It shouldn't have the tag field.

Instead of editing the value, have the value explicitly defined as before so it's more readable. Maybe call it `CONFIG_0_V0` for the V0 test